src/utils/style-sheet-utils.js

```diff
@@ -31,7 +31,12 @@ export const allCSSRulesAsArray = function(styleSheets) {
       try {
         if (styleSheet.cssRules) {
           // Convert the list of rules into an array.
-          const rulesAsArray = [].slice.call(styleSheet.cssRules);
+          const rulesAsArray = [].slice.call(styleSheet.cssRules)
+            .filter(rule => (
+              rule.type === CSSRule.STYLE_RULE ||
```

Comment:
By adding this filter, does the function still do what its name claims it does? The function name is `allCSSRulesAsArray` and the docs at the top say "Given a list of styleSheet-like objects, it returns an array of CSSStyleRule objects".

Reply:
@tchaffee The function name does not match the description initially, but now it does what the description says more precisely. It returns array of CSSStyleRule objects and CSSMediaRule objects with CSSStyleRule objects inside.

Let's rename it as allStyleRulesAsArray for example.